tests/Transfer/Commons/Tests/Doctrine/DbalAdapterTest.php

```diff
@@ -41,7 +41,7 @@ public function testHandle()
         $this->assertEquals(
             array(
                 array(
-                    'id' => 1,
+                    'id'   => 1,
```

Comment:
Totally agree, it felt wierd just doing it. Also, I was not aware of the fact the PRs are updated on the fly, when I push to the forked repo. With that In mind, this complain should, as you say, be solved elsewhere.